studio/components/interfaces/Settings/Logs/UpgradePrompt.tsx

```diff
@@ -27,33 +32,13 @@ const UpgradePrompt: React.FC<Props> = ({ projectRef, from }) => {
 
```

Comment:
All this logic can be extracted out to a utility function

Reply:
Cleaned this up and moved the logic out to a helper. I'll ping you on Monday and walk you through it rather have you comb through all of it again. Thanks for the review!